experiments-app/src/screens/ScrollViewEvents.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {
+  StyleSheet,
+  Text,
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+} from 'react-native';
+import { buildEventLogger } from '../utils/helpers';
+
+const handleOnMomentumScrollBegin = buildEventLogger('onMomentumScrollBegin');
+const handleOnMomentumScrollEnd = buildEventLogger('onMomentumScrollEnd');
+const handleOnScroll = buildEventLogger('onScroll');
+const handleOnScrollBeginDrag = buildEventLogger('onScrollBeginDrag');
+const handleOnScrollEndDrag = buildEventLogger('onScrollEndDrag');
+const handleOnScrollToTop = buildEventLogger('onScrollToTop');
+
+export function ScrollViewEvents() {
+  return (
+    <SafeAreaView style={styles.container}>
```

Comment:
Could you also add similar experiment for flat list. We would want to support both of these.